checkpatch.sh

```diff
@@ -164,8 +164,8 @@ function clang_style()
     local c="${options[cfg]}"
     echo "{ $(sed -e 's/#.*//' -e '/---/d' -e '/\.\.\./d' "$c" | tr $'\n' ,) }"
 }
-function clang_format() { clang-format --style="$(clang_style)" "$@"; }
-function clang_format_diff() { clang-format-diff --style="$(clang_style)" "$@"; }
+function clang_format() { clang-format -style="$(clang_style)" "$@"; }
```

Comment:
but `clang-format` has `--style`, although works both, ok

Reply:
Oh sorry, I overlooked...
Please feel free to revert this line.
Thanks for reviewing.